packages/flutter_tools/templates/app/.gitignore.tmpl

```diff
@@ -39,3 +39,7 @@ app.*.symbols
 
 # Obfuscation related
 app.*.map.json
+
+**/android/app/debug
```

Comment:
This will ignore every single folder that matches this pattern. Instead, you should only ignore the exact folders, like `/android/app/debug`. 

Also please leave a comment describing what this is for